resources/sdk/purecloudjavascript/templates/ApiClient.mustache

```diff
@@ -1163,6 +1194,61 @@ class ApiClient {
 		return url;
 	}
 
+    /**
+    	 * Builds http client.
+    	 * @returns {String} The http client default or the one that is injected.
+    	 */
+    	buildHttpClient() {
+    		if (this.httpClient) {
+    		    return this.httpClient;
+    		} else {
+    		    return this.defaultClient();
+    		}
+    	}
+
+        buildRequestOptions(url, method, headers, data, params) {
+             var request = new RequestOptions();
+             request.setMethod(method);
+             request.setUrl(url);
+             if (data) {
+                 request.setData(data);
+             }
+
+             if (params) {
+               request.setParams(params);
+             }
+
+             request.headers =  headers;
+             return request;
+        }
+
+         defaultClient() {
+                // Initialize proxyAgent as null
+                let proxyAgent = null;
+
+                // If agentOptions are provided, create a new https.Agent with those options
```

Comment:
At this time, it appears that we can either set certificates or set a proxyUrl (the legacy use of proxy). It would be necessary to find the necessary setting/setup/code to have both certificates and proxy url (if it makes sense), and if not, document in README that it is one or the other.

Reply:
updated the readme and addressed.